internal/provider/gce/disks_test.go

```diff
@@ -464,7 +500,7 @@ func (s *volumeSourceSuite) TestDescribeVolumes(c *gc.C) {
 	c.Check(err, jc.ErrorIsNil)
 	c.Assert(res, gc.HasLen, 1)
 	c.Assert(res[0].Error, jc.ErrorIsNil)
-	c.Assert(res[0].VolumeInfo.Size, gc.Equals, uint64(1024))
+	c.Assert(res[0].VolumeInfo.Size, gc.Equals, gce.GibToMib(10))
```

Comment:
Just use a literal here. No need to call the internal conversion. We know what the expected value should be